mods/ts/rules/civilian-structures.yaml

```diff
@@ -1306,6 +1252,7 @@ GASPOT:
 	Buildable:
 		Queue: Defense
 		Prerequisites: ~disabled
+		Description: Civilian Building
```

Comment:
Can we please finally remove `Buildable` from this and `GALITE`?  They are map props with zero reason to exist in the production sidebar when cheats are enabled.